### ISSUE TITLE
Issue loading a PDB that has Nucleic Acid Chains

### ISSUE THREAD

**Issue by zachcp** [douweschulte/pdbtbx#130]
Hi Team PDBTBX,

I've hit an issue opening a PDB that has a nucleic acid component.  Do you have any recommendations on how to work around or fix this issue? 

much obliged,
zach cp

```rust
// pdb == 1BC8.pdb
let (pdb, _) = pdbtbx::open(pdb_path).expect("A PDB  or CIF file");

```

```rust
A PDB  or CIF file: [LooseWarning: SEQRES inconsistent residues
    ╷
342 │ SEQRES   1 A   10   DT  DA  DC  DC  DG  DG  DA  DA  DG  DT
    ·                    ─── ─── ─── ─── ─── ─── ─── ─── ─── ───
    ╰SEQRES definition
```

![image](https://github.com/user-attachments/assets/f700ee29-c9b3-48d1-afaa-a9d404b69f99)

**Comment by zachcp** [comment]
Draft PR with the offending PDB is here: #131

**Comment by zachcp** [comment]
Note: I don't run into this when using the CIF version so I am unblocked on my end. 

Thank you again fro your great crate!

**Comment by douweschulte** [comment]
The issue seems to highlight that the information in the file is internally inconsistent, if this really is the case then the file itself is broken in some way (it might be edited without updating the SEQRES section for example). Still it is nice to have a nucleic acid PDB around as I did not encounter them that much yet, pdbtbx has mostly been protein centric.

It is good to hear that loading the CIF file worked, additionally this is only a `LooseWarning` so setting the `StrictnessLevel` when using the [`ReadOptions`](https://docs.rs/pdbtbx/latest/pdbtbx/struct.ReadOptions.html) to open the file would ignore this error. Control over the errors to ignore is given because there are many pdb files out in the wild that are incorrect given the formal specification but still provide unambiguous information.
